### DISCUSSION TITLE
RPiR Re-installation

### DISCUSSION THREAD

**Discussion by BakariMtili** [IBAHCM/RPiR#162] [Archive] [answered]
Hello @richardreeve @soniamitchell I am having issues with reinstalling RPiR. Its bringing this ' (converted from warning) installation of package ‘fastmap’ had non-zero exit status

In addition: There were 44 warnings (use warnings() to see them)'
Can I kindly get some assistance?

**Comment by soniamitchell** [comment, marked as answer]
Hi @2500251M, does it ask you to remove a `00LOCK` directory? If so, remove it and try `install.packages('fastmap')` before trying to install RPiR again.. 

If not, please restart your R session and try `install.packages('fastmap')` before trying to install RPiR again. 

For more details take a look here https://ibahcm.github.io/RPiR/articles/pages/install_RPiR.html#potential-problems.

If none of that works, could you take a screen shot of your error?

**Reply by BakariMtili** [reply]
It is taking long more than i anticipated.. i will keep you updated

**Reply by BakariMtili** [reply]
We did it, its all working now. Thank you
